infrastructure-playbooks/lv-create.yml

```diff
@@ -0,0 +1,96 @@
+- name: creates logical volumes for the bucket index or fs journals on a single device and prepares for use of osd_scenario=lvm. 
+  become: true
+  hosts:
+  - osds
+
+  vars:
+    logfile: |
+      Suggested cut and paste under "lvm_volumes:" in "group_vars/osds.yml" for configuring with osd_scenario=lvm
+      -----------------------------------------------------------------------------------------------------------
+      {% for lv in nvme_device_lvs %}
+        - data: {{ lv.lv_name }}
+          data_vg: {{ nvme_vg_name }}
+          journal: {{ lv.journal_name }}
+          journal_vg: {{ nvme_vg_name }}
+      {% endfor %}
+      {% for hdd in hdd_devices %}
+        - data: {{ hdd_lv_prefix }}-{{ hdd.split('/')[-1] }}
+          data_vg: {{ hdd_vg_prefix }}-{{ hdd.split('/')[-1] }}
+          journal: {{ hdd_journal_prefix }}-{{ hdd.split('/')[-1] }}
+          journal_vg: {{ nvme_vg_name }}
+      {% endfor %}
+
+  tasks:
+
+  - name: include vars of lv_vars.yaml
+    include_vars:
+      file: lv_vars.yaml
+    failed_when: false
```

Comment:
@leseb I'd say no because we don't want to restrict users ability to load configuration from other sources that ansible allows for. Requiring this file would also complicate how the CI tests run.

Reply:
Sounds good to me.